### DISCUSSION TITLE
Plugin: pixi-layers

### DISCUSSION THREAD

**Discussion by Flippsor** [pixijs/pixijs#10459] [v8 General]
Really love all changes in v8 done so far! Pixi Layers is the only thing which is missing for us to migrate to v8.

In the v8 Migration Guide [](https://pixijs.com/8.x/guides/migrations/v8) it is mentioned:

> Pixi layers (rather than migrating this, we will likely incorporate it directly into PixiJS v8 as a feature)

Are there already any plans if and when pixi-layers will be integrated into v8? Or maybe is there a quick workaround?

**Comment by shirecoding** [comment]
Just wondering is there any updates on this? would pixi-layers like functionalitly even in the pipeline at this point?

**Comment by StelyanBaltov** [comment]
Hello, I'm interested on an update on that topic as well. Pixi layers not being ported for pixi 8 is real showstopper for migrating.

**Reply by shirecoding** [reply]
How easy do you think it is possible to port over layers to the new architecture? As im not sure how much has changed if it is even possible

**Reply by StelyanBaltov** [reply]
Sadly I'havent had the time to go over the implementation details to estimate that. It came across my mind to port it myself, but not sure how complex it would be to achieve that. I was hoping on some answer from the PIXI guys because pixi v8 seems really solid, but for us, pixi layers is core functionality that we need

**Reply by shirecoding** [reply]
im too far into pixi8 and now just realized i need layers 💀

**Comment by alvov-evo** [comment]
Related discussion: https://github.com/pixijs/pixijs/discussions/11064
